TurkishFilter/sections/specific.txt

```diff
@@ -65,6 +65,7 @@ fullfilmcidayim.com,dizispeed.com,babafullfilmizle.com###adStarter
 /service/advertchannels?$domain=urfanatik.com|dirilispostasi.com|turkuazgazetesi.net|yenialanya.com|bugunkocaeli.com.tr|cagdaskocaeli.com.tr|mansetaydin.com|enkocaeli.com|kocaelikoz.com|kocaelibarisgazetesi.com|seskocaeli.com
 dirilispostasi.com,turkuazgazetesi.net,yenialanya.com,bugunkocaeli.com.tr,cagdaskocaeli.com.tr,mansetaydin.com,seskocaeli.com,kocaelibarisgazetesi.com,kocaelikoz.com,enkocaeli.com##.block-pano > div.us
 !
+||dizirun.co/megapari.mp4
```

Comment:
I don't see the request with AdGuard Turkish enabled. Is it needed?

Reply:
@zloyden I can see the request wih AdGuard Turkish enabled 
![image](https://user-images.githubusercontent.com/35523759/165055088-4c676dc3-b204-416f-8a78-e2a60aed1628.png)

Reply:
I see with Turkish IP.